.github/CODEOWNERS

```diff
@@ -177,6 +177,10 @@ CHANGELOG*
 /x-pack/metricbeat/docs/ # Listed without an owner to avoid maintaining doc ownership for each input and module.
 /x-pack/metricbeat/module/activemq @elastic/obs-infraobs-integrations
 /x-pack/metricbeat/module/airflow @elastic/obs-infraobs-integrations
+/x-pack/metricbeat/module/aws @elastic/obs-ds-hosted-services
+/x-pack/metricbeat/module/awsfargate @elastic/obs-infraobs-integrations
+/x-pack/metricbeat/module/azure @elastic/obs-ds-hosted-services
+/x-pack/metricbeat/module/azure/billing @elastic/obs-infraobs-integrations
```

Comment:
Should we add /x-pack/metricbeat/module/gcp also?

Reply:
You're right, I missed GCP.

According to the [spreadsheet](https://docs.google.com/spreadsheets/d/1vTK7XUwcItJRfr3LcYcWjjX7Y3Hf7QiA7gpHwMh3YmQ/edit#gid=1613284100) (my apologies for the private document), the ownership of the GCP metricsets is spread across several teams. 

I'm updating the CODEOWNERS file to match the spreadsheet content.